examples/hosts/iframe-host/src/inner.ts

```diff
@@ -51,8 +51,14 @@ async function loadContainer(
     const documentServiceFactory = await InnerDocumentServiceFactory.create(innerPort);
     const urlResolver = await InnerUrlResolver.create(innerPort);
 
-    const module = { fluidExport: TodoContainer };
-    const codeLoader = { load: async () => module };
+    const load = async (): Promise<IFluidModuleWithDetails> => {
+        return {
+            module: { fluidExport: TodoContainer },
+            details: { package: "no-dynamic-package", config: {} },
```

Comment:
@ChumpChief seems like we are using fake proposals often throughout the codebase. Should this eventually go away? It doesn't need to be this PR.